model/instance/manager.go

```diff
@@ -16,6 +16,9 @@ const (
 	ManagerPremiumURL
 	// ManagerBlockedURL is the kind for a redirection of a blocked instance.
 	ManagerBlockedURL
+	// ManagerFeatureSetsURL is the kind for an API call to list the feature
+	// flags associated to some feature sets.
+	ManagerFeatureSetsURL
```

Comment:
I think there is a problem here as we already discussed earlier. manager_url is currently used to return a customer-accessible URL to manager frontend

To access API from the stack we have a `cloudery:<context>:url` configuration in `cozy.yml`

Can you check that you use the right endpoint ?

Reply:
Good catch, it is not the right endpoint. I'm going to fix it.